schemas/context/pushdetail.example.1.json

```diff
@@ -0,0 +1,7 @@
+{
+    "xdm:appId": "75eafb7e-fa44-4514-86fc-221e32c5aef9",
+    "xdm:pushToken": "99156313-c9df-4e54-9c6c-5740f940c3ca",
```

Comment:
Modify the example with corresponding attribute/property described in JSON schemas